zookeeper/uninstalled.sls

```diff
@@ -0,0 +1,25 @@
+{%- from 'zookeeper/settings.sls' import zk with context -%}
+{%- from 'zookeeper/map.jinja' import zookeeper_map with context -%}
+
+zk-directories-removed:
+  file.absent:
+    - names:
+      - /var/run/zookeeper
+      - /var/lib/zookeeper
+      - /var/log/zookeeper
+      - {{ zk.real_home }}
+      - {{ zk.alt_home }}
+      - /etc/init.d/zookeeper
```

Comment:
I think this is already covered with `{{ zookeeper_map.service_script }}` item below.